src/cljs/rems/table.cljs

```diff
@@ -94,20 +140,38 @@
                               (s/optional-key :sort-column) s/Keyword
                               (s/optional-key :sort-order) (s/enum :asc :desc)
                               (s/optional-key :set-sorting) Applicable}
-   (s/optional-key :filtering) {(s/optional-key :filters) {s/Keyword s/Str}
+   (s/optional-key :filtering) {(s/optional-key :filters) s/Str
                                 (s/optional-key :show-filters) s/Bool
                                 (s/optional-key :set-filtering) Applicable}
    :id-function Applicable
    :items [s/Any]
    (s/optional-key :class) s/Str})
 
+(defn- filter-view [filtering]
+  (let [{:keys [filters set-filtering]} filtering]
+    [:div
+      [:input
```

Comment:
This input should be made full width. Also it doesn't align well with the search button.